templates/kube-oidc-proxy.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: kube-oidc-proxy
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: kube-oidc-proxy
+  annotations:
+    appversion.kubeaddons.mesosphere.io/dex-k8s-authenticator: "v1.1.0"
+    values.chart.helm.kubeaddons.mesosphere.io/dex-k8s-authenticator: "*"
```

Comment:
This should be a link, we're not trying to keep these annotations around long-term but at least for now let's make sure we add them.